sample-library/build.gradle

```diff
@@ -16,3 +16,9 @@ dependencies {
   implementation deps.appCompat
   testImplementation deps.junit
 }
+
+tasks.withType(JavaCompile) {
+  options.compilerArgs << "-AdeepLink.incremental=true"
```

Comment:
This is not in line with the documentation added in the `README.md` any reason?

Reply:
I have changed it to the same format as the `READ.md`. Turns out gradle has many different ways to do this :)

Reply:
Yeo, should just be consistent. :-)